apps/cookbook/src/app/showcase/card-showcase/card-showcase.component.html

```diff
@@ -125,17 +125,41 @@ <h3>Setting background image from CSS with media queries & linear gradient</h3>
     </cookbook-example-viewer>
   </p>
 
+  <h2>Card with item disclosure in card header</h2>
+  <p>
+    This example uses the <code>--kirby-card-padding-top</code> custom css property to set the
+    padding top in card content area to <code>0px</code>. Thereafter a item with disclosure is
+    placed in the card header. The Card header input property <code>hasPadding</code> is set to
```

Comment:
Should Card be starting with an uppercase? you are inconsistent on this

Reply:
fixed